tsconfig.json

```diff
@@ -15,6 +14,6 @@
     "strictNullChecks": true,
     "strictBuiltinIteratorReturn": false
   },
-  "include": ["./src/**/*.ts", "./test/**/*.ts"],
+  "include": ["./src/**/*.ts", "./test/**/*.ts", "eslint.config.mjs"],
```

Comment:
is file `eslint.config.mjs`  needed for build process? To me it looks like linter config